cli/command/container/completion.go

```diff
@@ -0,0 +1,94 @@
+package container
+
+import (
+	"github.com/docker/cli/cli/command/completion"
+	"github.com/docker/docker/api/types/container"
+	"github.com/moby/sys/signal"
+	"github.com/spf13/cobra"
+)
+
+// allLinuxCapabilities is a list of all known Linux capabilities.
+//
+// This list was based on the containerd pkg/cap package;
+// https://github.com/containerd/containerd/blob/v1.7.19/pkg/cap/cap_linux.go#L133-L181
+//
+// TODO(thaJeztah): add descriptions, and enable descriptions for our completion scripts (cobra.CompletionOptions.DisableDescriptions is currently set to "true")
+var allLinuxCapabilities = []string{
```

Comment:
Looks like containerd/pkg/cap maintains a list of these: https://github.com/containerd/containerd/blob/78d3e205a51ec101f775a43bee6f4fdd8fc6b22b/pkg/cap/cap_linux.go#L133-L187

Perhaps we could help extracting this from `pkg` to a separate repo that we could consume?

(not a blocker for this PR though)

Reply:
> Looks like containerd/pkg/cap maintains a list of these:

Hehe, yes, look at the comment above; that's where I took the list from 😄.

We _could_ consider moving such things to a separate repo (or to be in a smaller module); one tricky bit there is that these are linux capabilities, so if we want to use them cross-platform (in our case, the CLI may be running on Windows), at least we'd have to make clear that they only apply to Linux, so we can't have that code be platform-specific at compile time (`_linux.go` e.g.).

Reply:
Oh right, I didn't read the comment 🤦🏻 😄

Reply:
Yeah, we can't use it in the current form, but would be nice to avoid having to maintain this list by ourselves in future.

Reply:
Yeah agreed. For this one I decided that it's not _critical_ for the list to be up-to-date (even more accurate; completion may provide capabilities that are not supported by the daemon host if it's running an older kernel version), but for completion that should be mostly "OK"; the user doesn't _have_ to use the completion, and can still type something else.

For the `--cap-add` and `--cap-rm` cases, there's also the case that it would provide suggestions for capabilities that are either already set by default, or not present (both of which should be a no-op).

So the most _accurate_ thing to do would be to have API endpoints for these (i.e., let the daemon provide options that can be used) that are specifically crafted for purpose of completion (as lightweight as possible), but even if they're lightweight, there may still be some latency when connected to a remove machine, so that's something we should explore and consider pros/cons.